src/components/DocumentationTopic/DocumentationNav/Hierarchy.vue

```diff
@@ -137,9 +137,11 @@ export default {
   computed: {
     parentTopics() {
       return this.parentTopicIdentifiers.map((id) => {
-        const { title, url } = this.references[id];
+        const { title, url } = this.references[id] || {};
```

Comment:
Instead of simply not crashing in this situation, maybe we should additionally do something like `console.warn` to indicate that there was a missing reference since this is an unexpected/bad state where something went wrong with the compiler or otherwise with the data?

Reply:
Will add a `console.error` here. In the future we may want to have a more refined user notification process, but that should do for now.